src/page/generate/GenerateDocumentHistoryPage.tsx

```
@@ -2,124 +2,314 @@
-                        });
-                    } else if (token.type === 'paragraph') {
-                        return new Paragraph({
-                            children: [new TextRun({ text: token.text })],
-                            spacing: { after: 200 }
-                        });
+  const [documents, setDocuments] = useState<Document[]>([]);
+  const [viewedDocumentString, setViewedDocumentString] = useState<
+    string | null
+  >(null);
+  const [isSidebarVisible, setIsSidebarVisible] = useState(true);
+  const [documentObject, setDocumentObject] = useState<any>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)